### PR TITLE
Add support for generic installers

### PR DESCRIPTION
**Pull request by chase-qi**
Introduce the following new vars to support generic installers.
    * erp_installer_distro
    * erp_image_remote_paths

Signed-off-by: Chase Qi <chase.qi@linaro.org>

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
 
 
 # Used to download builds
-erp_debian_installer_download_urls:
+erp_installer_download_urls:
   staging: "http://snapshots.linaro.org/96boards/reference-platform/components/debian-installer-staging/"
   stable: "http://snapshots.linaro.org/96boards/reference-platform/components/debian-installer/"
 
+# Used to download build images
+erp_image_remote_paths:
```
Comment (danrue) [inline defaults/main.yml:22]: Does this var need to be added to README?

**Review by danrue — APPROVED** [review]

File: defaults/main.yml (hunk 1 above)
Comment (chase-qi) [reply defaults/main.yml:22]: I have updated the patch to add the var to README.

BTW, erp_installer_download_urls_api and erp_installer_download_urls are also missing in README. I will try to add them with another PR.

**Comment by chase-qi** [comment]
Dan, thanks for the review! I am going to 'ship it' :)

**Comment by danrue** [comment]
:shipit: :smiley:

File: defaults/main.yml (hunk 1 above)
Comment (chase-qi) [reply defaults/main.yml:22]: All vars added to README.

**Review by chase-qi — COMMENTED** [review]
ha, more emojis

**Pull request merged**